skeleton_scary_game.py

```diff
@@ -0,0 +1,329 @@
+            if need_help.lower() == "y":
+                print("\nPick an item you would like to craft.")
+                pick_an_item = input(str("1.----->   Torch \n2.----->   Pistol \n==> "))
+                print()
+                if pick_an_item == "1":
+                    self.father.create_torch(self.item.users_bagpack)
+                elif pick_an_item == "2":
+                    self.father.create_gun(self.item.users_bagpack)
+                else:
+                    print("You did not pick one of the options given!")
+            elif need_help.lower() == "n":
+                print("You have these items on your bagpack:", self.item.users_bagpack)
+                print()
+
+            # The user will be asked if they would like to use an item, if they pick y then they will choose what item,
+            # If they pick n then they will n, it will continue on with the game.
+            try:
+                use_an_item = input(str("\nWould you like to use an Item (y/n) => "))
+                if use_an_item.lower() != 'y' and use_an_item.lower() != 'n':
+                    raise ValueError("PLEASE ENTER y or n!")
+            except ValueError as d:
+                print(str(d))
+                continue
+
+
+            # User input on what item they would like to use, and from there call in the function of the father class.
+            if use_an_item == 'y':
+                print("\nWhich Item would you like to use?\n1. ---> Flashlight\n2. ---> Torch\n3. ---> Gun")
+                item_usage = input(str("==> "))
+                print()
+                if item_usage == "1":
+                    self.father.use_item(self.item.users_bagpack, demon_path)
+                elif item_usage == "2":
+                    self.father.use_torch(self.item.users_bagpack, item_path)
+                elif item_usage == "3":
+                    if "Deagle" in self.item.users_bagpack and "Bullets" in self.item.users_bagpack:
+                        self.father.use_pistol(self.item.users_bagpack)
+                        a = 1
+                    else:
+                        print("You do not have all the gun parts")
+                else:
+                    print("That is not an option.\n")
+
+            # Display the available paths
+            for i, path in enumerate(self.paths):
+                print(f"{i + 1}. {path}\n")
+
+            # User picks their path
+            loop_count += 1
+
+            # This if will run if the user did not use the gun to kill the demon
+            if a == 0:
+                print("Becareful and choose the right path\n")
+                while True:
+                    try:
+
+                        # If the user picks a path number that is not valid, they have to re-pick
+                        user_choice = int(input(f"Tunnel #{loop_count} "
+                                                f"Walk your path => "))
```

Comment:
Instead of "Tunnel" can you change it to "Turn", and instead of "Walk" can you change it to "Choose". One of the turns I got confused between the Tunnel # and what the flashlight said.